test/action_controller/json_api/linked_test.rb

```diff
@@ -51,9 +51,9 @@ def render_resource_with_nested_include
             render json: @post, include: [comments: [:author]], adapter: :json_api
           end
 
-          def render_resource_with_nested_has_many_include
+          def render_resource_with_nested_has_many_include_wildcard
             setup_post
-            render json: @post, include: 'author.roles', adapter: :json_api
+            render json: @post, include: 'author.*', adapter: :json_api
           end
```

Comment:
Actually, I don't believe there should be more integration tests for this PR. We should just thoroughly test the `expand_includes` method.